docs/examples/prototypes.md

```diff
@@ -1,5 +1,57 @@
 # Prototypes 
 
-## Available Prototypes
+You would like to explore the applications that you can build with Amadeus Self-Service APIs? 
```

Comment:
This line was changed to [this](https://github.com/amadeus4dev/developer-guides/pull/32/commits/260485ec08ebe6ce63bfeaad6f93f30c048c213f#diff-6986952fc0dba8e9ebdb091025c5aaa59fc442113a9dfb056a111e01a3155621R3), but in this commit here I see it revereted